src/components/radios/index.md.njk

```diff
@@ -25,13 +27,13 @@ There are two ways to use the Radios component. You can use HTML or, if you are
 
 When there are more than two options, radios should be stacked, like so:
 
-[Example of stacked radios]
+{{ example({group: 'components', item: 'radios', example: 'default', html: true, nunjucks: true, open: true}) }}
```

Comment:
I will extend the example tab container id to allow the same example being placed on the same page with and without code blocks - discussed this problem with @dashouse offline.